Gems/Atom/Feature/Common/Assets/Passes/Mobile/Pipeline.pass

```diff
@@ -87,6 +87,20 @@
                         }
                     }
                 },
+                {
+                    "Name": "OutlineGatherPass",
```

Comment:
Is the forward pass writing to stencil buffer with a mask and then this pass uses that mask to write a color into the RT?

Reply:
yup!